tests/constraints.in

```
@@ -1,5 +1,8 @@
 # This constraints file contains pins for the stable, tested versions of sphinx
 # and antsibull-docs that production builds rely upon.
+# This constraint file also pins other versions for which there are known limitations.
 
 sphinx == 7.2.5
 antsibull-docs == 2.16.2  # currently approved version
+sphinx-rtd-theme >= 2.0.0  # Fix 404 pages with new sphinx -- https://github.com/ansible/ansible-documentation/issues/678
```

How about separating the two blocks:
```suggestion

sphinx-rtd-theme >= 2.0.0  # Fix 404 pages with new sphinx -- https://github.com/ansible/ansible-documentation/issues/678
```